Lombiq.Vsix.Orchard/Commands/InjectDependencyCommand.cs

```diff
@@ -90,21 +86,15 @@ private async void MenuItemCallback(object sender, EventArgs e)
 
                     var result = _dependencyInjector.Inject(
                         dte.ActiveDocument,
-                        injectDependencyDialog.GetDependencyInjectionData());
+                        injectDependencyDialog.DependencyInjectionData);
 
                     if (!result.Success)
                     {
-                        switch (result.ErrorCode)
-                        {
-                            case DependencyInjectorErrorCodes.ClassNotFound:
-                                DialogHelpers.Warning(
-                                    "Could not inject dependency because the class was not found in this file.",
-                                    injectDependencyCaption);
-                                break;
-                            default:
-                                DialogHelpers.Warning("Could not inject dependency.", injectDependencyCaption);
-                                break;
-                        }
+                        DialogHelpers.Warning(
+                            result.ErrorCode == DependencyInjectorErrorCodes.ClassNotFound ?
+                            "Could not inject dependency because the class was not found in this file." :
+                            "Could not inject dependency.",

```

Comment:
Ezt is pedig asszem egy analyzernek jeleznie kellene:
```suggestion
                            result.ErrorCode == DependencyInjectorErrorCodes.ClassNotFound
                                ? "Could not inject dependency because the class was not found in this file."
                                : "Could not inject dependency.",

```